src/components/UserInfo/UserInfo.tsx

```diff
@@ -2,0 +1,17 @@
+import React from 'react';
+
+import './UserInfo.scss';
+
+import { User } from '../../types';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
```

Comment:
[CODE KNOWLEDGE] - Since the `user` prop is of type `User` and not optional, you do not need to use the optional chaining operator `?.` when accessing `user.email`. It is safe to assume that `user` will always be an object with an `email` property.